python/lsst/sims/maf/db/opsimDatabase.py

```diff
@@ -154,7 +154,7 @@ def fetchLatLonHeight(self):
             site = Site(name='LSST')
             lat = site.latitude_rad
             lon = site.longitude_rad
-            height = site.elev
+            height = site.height
```

Comment:
why was this even working before?? gah. Thanks.

Reply:
gotta love interpreted languages that only raise errors on code that actually gets run =)